dashboards-yaml-reference_5ba340d42c7d3a16370f5104.md

```diff
@@ -136,10 +136,11 @@ The supported properties in `filters` for a workflows list are the following:
     of the GitHub repository of a Semaphore 2.0 project. Note that the value of the `branch` property
     should be an exact match and that there is currently no support for regular expressions
     in the `branch` property.
-* **github_uid** (optional): The `github_uid` property allows you to display workflows
-    that were initiated by the specified GitHub username. There exist a special
-    value for `github_uid`, which is `'{{gitgub_uid}}'` that is automatically
-    expanded to the GitHub username of the current user.
+* **github_uid** (optional): The `github_uid` property allows you to display
+    workflows that were initiated by the GitHub user with the same GitHub user
+	ID as the value of the `github_uid` property. There exists a special value
+	for the `github_uid` property, which is `'{{github_uid}}'`, that is
+	automatically expanded to the GitHub user ID of the current user.
```

Comment:
The `github_uid` property enables you to list workflows which were initiated by specific user account on GitHub. The value of `github_uid` should correspond to the ID of the user account on GitHub for which to list workflows. The value can be found by visiting corresponding URL https://api.github.com/users/<github-username>.